src/WebformCivicrmPostProcess.php

```diff
@@ -718,6 +718,9 @@ private function createContact($contact) {
     if ($params['contact_type'] == 'Individual' && empty($params['first_name']) && empty($params['last_name'])) {
       $params['display_name'] = $params['sort_name'] = empty($params['nick_name']) ? $contact['email'][1]['email'] : $params['nick_name'];
     }
+    if (isset($params['image_URL'])) {
+      \CRM_Contact_BAO_Contact::processImageParams($params);
+    }
     $result = $utils->wf_civicrm_api('contact', 'create', $params);
     return wf_crm_aval($result, 'id', 0);
   }
```

Comment:
We've been working hard to remove all BAO function calls (replacing them with API calls). There are only few left to go! So we should not be adding any new ones. Can you replace this with an API call?